tasks/manager.yml

```diff
@@ -23,6 +23,7 @@
     - ara-server
     - ara-web
     - database
+  when: ara_enable == true
```

Comment:
Use ``ara_enable|bool`` (ansible-lint: [601] Don't compare to literal True/False)

Reply:
Done